frontend/components/ClubEditPage/AdvisorCard.tsx

```diff
@@ -15,9 +17,36 @@ import BaseCard from './BaseCard'
 type Props = {
   club: Club
   schools: School[]
+  validateAdvisors?: (any) => void
```

Comment:
You can annotate this with `(valid: boolean) => void` or a similar type to be more accurate.